runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -105,6 +103,8 @@ public long append(@Nonnull Set<UUID> streamIDs, @Nonnull Object object,
                 : runtime.getSequencerView().nextToken(streamIDs, 1,
                 conflictInfo); // Token w/ conflict info
 
+        final ILogData ld = new LogData(DataType.DATA, object);
+        final boolean cp = ld.hasCheckpointMetadata();
```

Comment:
Also, I'm pretty sure this isn't going to do what you expect it to do... You're going to wrap the object, which is probably already a logData.. in another LogData. Instead you should check if the object is an instance of LogData